package.json

```diff
@@ -39,7 +39,7 @@
     "needle": "^2.4.0",
```

Comment:
typo in commit message

Reply:
also please explain the change slightly more thoroughly in the title, keep in mind these are the release notes our users will see:
```
upgraded docker plugin version (43e8022)
```
against something like
```
support scanning for OpenJDK binaries
```

Reply:
on it 👍